arrows/core/video_input_image_list.cxx

```diff
@@ -127,14 +136,25 @@ ::get_configuration() const
   vital::config_block_sptr config = vital::algo::video_input::get_configuration();
 
   config->set_value( "path", "",
-                     "Path to search for image file. "
-                     "If a file name is not absolute, this list of directories is scanned to find the file. "
-                     "The current directory '.' is automatically appended to the end of the path. "
-                     "The format of this path is the same as the standard "
-                     "path specification, a set of directories separated by a colon (':')" );
+    "Path to search for image file. "
```

Comment:
Wrong indent here; please either keep things aligned to the `(` or break immediately after the `(` (and then indent one more level than the previous line).